TempleEight/docs/reference/templefile-primitives.md

```diff
@@ -4,13 +4,13 @@ title: Templefile Primitives
 sidebar_label: Templefile Primitives
 ---
 
-Templefile supports 8 primitive types, roughly aligned with those found in `SQL` style languages. Any property of a service can be of any of these types. 
+Templefile supports 8 primitive types, roughly aligned with those found in `SQL` style languages. Any property of a service can be of any of these types.
```

Comment:
Wonder if we should use attribute rather than property?
```suggestion
Templefile supports 8 primitive types, roughly aligned with those found in `SQL` style languages. Any attribute of a service can be of any of these types.
```